### PR TITLE
Diagram

### PR DESCRIPTION
**Pull request by josuerushanika**
In this PR we did :

- [ ]  Create table based on  database diagram for a clinic.
- [ ] Create a file named schema_based_on_diagram.sql where we implement the database from the diagram.
- [ ]  Add the FK indexes.

### CODE REVIEW

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice schema file ✔️
- Good commit messages. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
@zieeco 
 I made a mistake and i deleted your review comment from my Vs code   can you return it back before i can submit the second review  please?

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    total_price  decimal,
+    invoice_id  int, 
+    treatment_id int,
+    FOREIGN KEY ( invoice_id) REFERENCES  invoices (id),
+    FOREIGN KEY ( treatment_id) REFERENCES treatments (id),
+);
+
+CREATE INDEX idx_treatment_id ON medical_histories_treatments (treatment_id);
+CREATE INDEX idx_medical_history_id ON medical_histories_treatments (medical_history_id);
+
```
Comment (zieeco) [inline schema_based_on_diagram.sql:43]: ### Adding the review again as you deleted my initial review comment
### Add the missing indexes on ALL THE FOREIGN KEYS


- Hey, :wave: Please, let's not forget also to the indexes on ALL THE FOREIGN KEYS  to meet the project requirement. see an example:

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
```
This should be added for ALL THE FOREIGN KEYS, that's just a few from the rest.

**Review by zieeco — COMMENTED** [review]
Hey @josuerushanika, kindly implement the changes in my the review as you've deleted my initial review. According to my initial review [summary](https://github.com/josuerushanika/Vet-Clinic-Database/pull/5#pullrequestreview-1390569717), you should implement them before you submit for another review from your dashboard.

**Review by m05e5 — APPROVED** [review]
**Hi @josuerushanika 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean Codebase
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**